src/index.html

```diff
@@ -11,12 +11,231 @@
+              </a>
+            </div>
+            <h3 class="category__title">All products</h3>
+          </article>
+
+          <article class="category">
+            <div class="category__photos">
+              <a href="#" class="category__link category__link__square">
+                <img class="category__photo" src="../src/images/categor3.svg"
+                alt="img">
+              </a>
+
+              <a href="#" class="category__link category__link__wide">
+                <img class="category__photo" src="../src/images/2-big.svg"
+                alt="img">
+              </a>
+            </div>
+            <h3 class="category__title">Audio</h3>
+          </article>
+
+          <article class="category">
+            <div class="category__photos">
+              <a href="#" class="category__link category__link__wide">
+                <img class="category__photo" src="../src/images/3-big.svg"
+                alt="img">
+              </a>
+
+              <a href="#" class="category__link category__link__square">
+                <img class="category__photo" src="../src/images/categor6.svg"
+                alt="img">
+              </a>
+            </div>
+            <h3 class="category__title">Accessories</h3>
+          </article>
+
+          <div class="category__box__link">
+            <a href="#" class=" link category__box__link__button">View all</a>
+          </div>
+        </div>
+      </section>
+
+      <!-- region recommended -->
+      <section class="about-us">
+        <img class="about-us__black__fone" src="../src/images/black-fone.svg"
+        alt="black">
+        <h2 class="section-title">About us</h2>
+
+        <div class="about-us__content">
+          <p class="about-us__paragraphs">We’re a London-based tech company on a mission to remove barriers between people and technology.</p>
+          <br>
+          <p class="about-us__paragraphs">
+            To do this, we’re crafting intuitive, flawlessly connected products that improve our lives without getting in the way.
+            <br>
+            No confusing tech-speak. No silly product names. Just artistry, passion and trust. And products we’re proud to share with our friends and family. Simple.
+          </p>
+          <br>
+          <p class="about-us__paragraphs">Because beautiful tech shouldn’t be complicated.</p>
+        </div>
+      </section>
+      <!-- region contacts -->
+      <section class="contacts">
+        <div class="contacts__box">
+          <h2 class="section-title">Contact us</h2>
+          <input class="contacts__name" type="text"
+          name="name"
+           placeholder="Name">
+
+          <input class="contacts__email" type="email"
+          name="email"
+           placeholder="Email">
+
+          <textarea class="contacts__message" name="message"
+          cols="30"
+           rows="10" placeholder="Message"></textarea>
+          <a class="link contacts__send" href="#">Send</a>
```

Comment:
Please, wrap these with the form and use the button (or input with `type=submit`) for the `Send`